### PR TITLE
Refactor/fix ivanalysis

### PR DESCRIPTION
**Pull request by cwfink**
Updated the `IVAnalysis` class to be compatible with the new `IBIS` and `DIDV` classes in `QETpy`. This PR also adds all the functionality to the `IVAnalysis` class that was in the branch https://github.com/ucbpylegroup/RQpy/tree/refactor/IVerror_prop 

I've re-analyzed SLAC PD2 data with this updated code and confirmed that all the functionality works.

### CODE REVIEW

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (slwatkins) [inline rqpy/plotting/_iv_didv_tools_plotting.py:750]: Perhaps we should replace the `+` with an ampersand `&`? I feel that the plus sign is a little colloquial for plots.
```suggestion
            label='Squid & Electronics Noise',
```

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (slwatkins) [inline rqpy/plotting/_iv_didv_tools_plotting.py:654]: Perhaps we should replace the `+` with an ampersand `&`? I feel that the plus sign is a little colloquial for plots.
```suggestion
            label='Squid & Electronics Noise',
```

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (slwatkins) [inline rqpy/plotting/_iv_didv_tools_plotting.py:168]: Does this labeling work with the ℓ and Ω characters? Or should we switch to their LaTeX versions (\ell, \Omega)

```suggestion
    axes[1].set_ylabel(r'$R_{N} + R_\ell$ [m$\Omega$]', fontsize=14)
```

Comment (slwatkins) [reply rqpy/plotting/_iv_didv_tools_plotting.py:168]: I know that Python 3 should support it, but sometimes the LaTeX compiler errors on the special characters in between $'s

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (cwfink) [reply rqpy/plotting/_iv_didv_tools_plotting.py]: added

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (cwfink) [reply rqpy/plotting/_iv_didv_tools_plotting.py]: added

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (cwfink) [reply rqpy/plotting/_iv_didv_tools_plotting.py]: changed this, and a few more places as well

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (slwatkins) [inline rqpy/plotting/_iv_didv_tools_plotting.py:153]: The `\text` function I don't think is supported in matplotlib LaTeX
```suggestion
    axes[0].set_xlabel(r'$V_\mathrm{bias} \, [\mu\mathrm{V}]$', fontsize = 14)
```

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (slwatkins) [inline rqpy/plotting/_iv_didv_tools_plotting.py:169]: The `\text` function I don't think is supported in matplotlib LaTeX
```suggestion
    axes[1].set_xlabel(r'$V_\mathrm{bias} \, [\mu\mathrm{V}]$', fontsize=14)
```

File: rqpy/plotting/_iv_didv_tools_plotting.py
Comment (slwatkins) [inline rqpy/plotting/_iv_didv_tools_plotting.py:21]: Whoops, I guess I added an extra return here!

```suggestion
```

**Review by slwatkins — APPROVED** [review]
After the small fixes, I haven't noticed any glaring issues. @cwfink I'll approve this for now - do you want to do one last analysis sweep check before merging?

**Comment by cwfink** [comment]
Found syntax error. Doing one final analysis sweep check, then will merge once everything checks out

**Comment by cwfink** [comment]
Found and fixed an issue with how the priors covariance matrix was constructed when doing the 3 pole priors fit.

File: rqpy/core/_iv_didv_tools.py
Comment (slwatkins) [inline rqpy/core/_iv_didv_tools.py:1400]:
```suggestion
            
```

Extra print statement?

**Pull request merged**
